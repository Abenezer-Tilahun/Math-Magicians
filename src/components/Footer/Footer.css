footer {
  text-align: center;
  padding: 1rem 1rem;
  background-color: #f5f5dc;
}

footer p {
  color: blueviolet;
}

footer a {
  color: blueviolet;
  margin-inline: 0.25rem;
}
