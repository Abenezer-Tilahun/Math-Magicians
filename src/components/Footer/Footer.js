import React from 'react';
import './Footer.css';

export default function Footer() {
  return (
    <footer>
      <p>This is a Math-Magicians Software Designed By Abenezer-Tilahun </p>
      <a href="https://github.com/Abenezer-Tilahun/Math-Magicians.git" target="_blank" rel="noopener noreferrer">
        Copyright @2022
      </a>
      .
    </footer>
  );
}
