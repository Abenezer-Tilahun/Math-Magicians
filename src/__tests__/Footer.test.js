import React from 'react';
import Renderer from 'react-test-renderer';
import Footer from '../components/Footer/Footer';

it('Renders correctly', () => {
  const tree = Renderer
    .create(<Footer />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
