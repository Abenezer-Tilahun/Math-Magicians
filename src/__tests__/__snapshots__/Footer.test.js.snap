// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<footer>
  <p>
    This is a Math-Magicians Software Designed By Abenezer-Tilahun 
  </p>
  <a
    href="https://github.com/Abenezer-Tilahun/Math-Magicians.git"
    rel="noopener noreferrer"
    target="_blank"
  >
    Copyright @2022
  </a>
  .
</footer>
`;
